django-stubs/db/models/query.pyi

```diff
@@ -41,7 +41,7 @@ class _BaseQuerySet(Generic[_T], Sized):
         hints: Optional[Dict[str, models.Model]] = ...,
     ) -> None: ...
     @classmethod
-    def as_manager(cls) -> Manager[Any]: ...
+    def as_manager(cls) -> Manager[_T]: ...
```

Comment:
Can you please also test it? We have a lot of bugs in QuerySet / Manager topics.
And we want to be sure it is as stable as possible.